contrib/confluentinc/confluent-kafka-go/kafka/option.go

```diff
@@ -82,3 +85,14 @@ func WithAnalyticsRate(rate float64) Option {
 		}
 	}
 }
+
+// WithCustomTag will cause the given tagFn to be evaluated after executing
+// a query and attach the result to the span tagged by the key.
+func WithCustomTag(tag string, tagFn func(msg *kafka.Message) interface{}) Option {
```

Comment:
Why do we need the word "custom"? It's just a regular tag. How about `WithTag` or `WithTagger`?
```suggestion
func WithCustomTag(key string, valueFn func(msg *kafka.Message) interface{}) Option {
```
Nit: Maybe it makes more sense to name them key/value

Reply:
I don't have a strong preference on naming. The current name `WithCustomTag` is picked for consistency with one existing [implementation](https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/contrib/jinzhu/gorm#WithCustomTag)

Reply:
Oh, right! Thanks for jumping in. I didn't realise we were already using that naming, in which case it's fine!